Alberto-todo-list/src/components/FormTodo.jsx

```diff
@@ -1,17 +1,17 @@
-import React, { useState } from "react";
+import { useState } from "react";
 
-const FormTodo = Ftodo => {
+const FormTodo = props => {
```

Comment:
important! respect this particular naming convention. 

-> the argument for the components should be named `props`.